src/main/java/org/symphonyoss/integration/jira/webhook/JiraWebHookIntegration.java

```diff
@@ -171,13 +172,13 @@ public void authorize(AuthorizationPayload authorizationPayload) throws Authoriz
 
     if (StringUtils.isBlank(temporaryToken) || StringUtils.isBlank(verificationCode)) {
       throw new JiraOAuth1Exception(MSG.getMessage(MSG_INSUFFICIENT_PARAMS),
-          MSG.getMessage(MSG_INSUFFICIENT_PARAMS_SOLUTION));
+          HttpStatus.BAD_REQUEST.value(), MSG.getMessage(MSG_INSUFFICIENT_PARAMS_SOLUTION));
```

Comment:
I don't think we should handle HTTP codes in this layer. This layer should be independent of protocols. It's better to extend JiraOAuth1Exception and create a meaningful Exception to express what you want here.